client/components/LearnerViewDirective.js

```diff
@@ -249,10 +249,20 @@ tie.directive('learnerView', [function() {
         $scope.submitCode = function(code) {
           SolutionHandlerService
             .processSolutionAsync(prompts[currentPromptIndex], code, language)
-            .then(appendFeedback);
+            .then(setFeedback);
         };
 
-        clearFeedback();
+        $scope.feedbackParagraphs = [
+          'Greetings!',
+          'This set of exercises focuses on string manipulation.',
```

Comment:
Good catch. I had considered doing that, actually, but in my experience it would be better to do that a bit later, when the language and data schema have stabilized a bit more. Otherwise there's a lot of interim churn/indirection and I don't think that is necessary at this stage. Ultimately, I would like to grab the "focus point" for the exercises from the data file, but we need a data-driven way to represent a set of exercises before that happens.

So, I'd prefer to pull this out when we start supporting these additional use cases (e.g. separate languages), or making plans to do so, but not before. Does that sound OK?